### ISSUE TITLE
Creation of a release

### ISSUE THREAD

**Issue by gilardh** [MyHush/hushpaperwallet#4]
It would be necessary to create a release for aarch64, armv7, linux, macOS and Windows.

**Comment by leto** [comment]
@gilardh nope, we can just make whatever we can. Most people use windows, then mac, then linux, and most people don't use aarch64 or armv7, but we can try to make those
